.github/hibernate-github-bot.yml

```diff
@@ -4,4 +4,29 @@ jira:
   ignore:
       # See the `build-dependencies` group in the Dependabot's configuration file
     - user: dependabot[bot]
-      titlePattern: "Bump the build-dependencies group with \\d+ updates?"
\ No newline at end of file
+      titlePattern: "Bump the build-dependencies group with \\d+ updates?"
+  ignoreFiles:
+    # Git
+    - ".git*"
+    - ".mailmap"
+    # Maven
+    - "mvnw*"
+    - ".mvn/"
+    # CI
+    - ".github/"
+    - "ci/"
+    - "Jenkinsfile"
+    - "*/Jenkinsfile"
+    - "*.Jenkinsfile"
+    - "*.sh"
+    # In-repo documentation
+    - "copyright.txt"
+    - "README.md"
+    - "MAINTAINERS.md"
+    - "CONTRIBUTING.md"
+    # Misc. build files
+    - "build/config/"
+    - "build/enforcer/"
+    - "build/jqassistant/"
+    - "build/reports/"
+    - "build/script/"
```

Comment:
There are also `build/configuration-properties-collector` and `build/container*`  that also fall under this category no?

Reply:
Right, will do